src/views/NutritionView.vue

```diff
@@ -283,6 +286,8 @@ export default {
         };
       }
       this.currentProductData = data;
+      this.nutritionData = await this.getNutritionData();
+      this.setNutritionData();
```

Comment:
Do you use or plan to use `this.nutritionData` somewhere else?

If no, It may be interesting to move the call to `this.getNutritionData();` inside `setNutritionData`